README.md

```diff
@@ -1 +1,14 @@
-# cocoapod-specs
\ No newline at end of file
+# The Bitmovin CocoaPods Repo
+
+This repository contains all public CocoaPods specifications provided by [Bitmovin](https://bitmovin.com).
+
+## Using
+
+1. Add `source 'https://github.com/bitmovin/cocoapod-specs.git'` to your `Podfile`.
+1. Run `pod repo update` to add the newly added source.
+1. Add the desired pod (e.g. `pod 'BitmovinPlayer', '3.0.0'`) to your `Podfile`.
+1. Install the pod using `pod install`.
+
+## Support
+
+If you have any questions or issues, or you require other technical support for our services, please login to your Bitmovin Dashboard at https://bitmovin.com/dashboard and create a new support case. Our team will get back to you as soon as possible 👍
```

Comment:
I'd link directly to the support case section in the dashboard:

```suggestion
If you have any questions or issues, or you require other technical support for our services, please login to your Bitmovin Dashboard and create a new support case at https://bitmovin.com/dashboard/support/cases/create?tab=player. Our team will get back to you as soon as possible 👍
```

Reply:
can do 👍 

_This was also copied from the public sample repo_

Reply:
4168ae8